scripts/generate_api_key_scope_enum/main.go

```diff
@@ -11,22 +12,41 @@ import (
 func main() {
```

Comment:
Did you have to remove the non-wildcard values from the output?

Reply:
No. The generator still appends every action-specific (`resource:action`) scope using `add(fmt.Sprintf(…))`, so those non-wildcard values remain in the output. We only added wildcard generation (`resource:*`) and deduplication in the new `add` helper.

Reply:
The output generated though, it generates the entire set right? You have to manually clean it up for the migration?